code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -33,6 +33,8 @@
 	var/check_records = TRUE //Does it check security records?
 	var/arrest_type = FALSE //If true, don't handcuff
 	var/russian = FALSE // If true, it uses russian voice lines
+	var/stuncount = 0 // The securitron will stun you until it gets tired of doing it
+	var/mob/lastStunned // Who was stunned last?
```

Comment:
```suggestion
	var/mob/living/carbon/lastStunned // Who was stunned last?
```
Just for the sake of avoiding potential bugs. Shouldn't change the functionality